geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientTestCase.java

```diff
@@ -729,6 +730,12 @@ public void run2() throws CacheException {
       }
     });
 
+    try {
```

Comment:
There seems to be some delay with the registering of interest. I'm not fond of this wait, but there isn't a better way I saw. Nothing to check for that would provide good feedback. I am open to suggestions.